eth/protocols/eth/protocol.go

```diff
@@ -111,9 +111,86 @@ func (p *NewBlockHashesPacket) Unpack() ([]common.Hash, []uint64) {
 	return hashes, numbers
 }
 
+// transactionEnvelope is an envelope over a transaction such that
+// out-of-protocol metadata can be gossiped to peers
+type transactionEnvelope struct {
+	Tx          *types.Transaction
+	Conditional *types.TransactionConditional
+}
```

Comment:
Do we have specs for this? We need other EL clients to be able to sync transactions with op-geth still, if users start running op-reth/op-erigon/op-nethermind more in production sequencing settings.